ray/README.md

```diff
@@ -0,0 +1,70 @@
+# Deploying Ray with Open Data Hub
+
+_WIP Docs: _
+
+Integration of [Ray](https://docs.ray.io/en/latest/index.html) with Open Data Hub on OpenShift. The ray operator and other components are based on https://docs.ray.io/en/releases-1.13.0/cluster/kubernetes.html
+
+## Components  of the Ray deployment
+
+1. [Ray operator](./operator/ray-operator-deployment.yaml): The operator will process RayCluster resources and schedule ray head and worker pods based on requirements.
+2. [Ray CR](./operator/ray-custom-resources.yaml):  RayCluster Custom Resource (CR) describes the desired state of ray cluster.
+3. [Ray Cluster](./cluster/ray-cluster.yaml): Defines an instance of an example Ray Cluster 
+
+
+## Deploy the RayCluster Components:
+
+Prerequisite to install RayCluster with ODH:
+
+* Cluster admin access
+* An ODH deployment
+* [Kustomize](https://kustomize.io/) 
+
+### Install Ray
+
+We will use [Kustomize](https://kustomize.io/) to deploy everything we need to use Ray with Open Data Hub. 
+
+#### Install the operator and custom resource 
+
+First use the `oc kustomize` command to generate a yaml containing all the requirements for the operator and the "raycluster" custom resource, then `oc apply` that yaml to deploy the operator to your cluster. 
+
+```bash
+$ oc kustomize deploy/odh-ray-nbc/operator > operator_deployment.yaml
+```
+```bash
+$ oc apply -f operator_deployment.yaml
+```
+
+#### Confirm the operator is running 
+
+```
+$ oc get pods 
+NAME                               READY   STATUS    RESTARTS        AGE
+ray-operator-867bc855b7-2tzxs      1/1     Running   0               4d19h
+
+```
+
+#### Create a ray cluster 
+
+
+```bash
+$ oc kustomize deploy/odh-ray-nbc/cluster > cluster_deployment.yaml
```

Comment:
same as before, the path `deploy/odh-ray-nbc/cluster` seems like a typo, looks like the cluster is at `ray/cluster/base`